core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -89,9 +89,10 @@ class KafkaRequestHandlerPool(val brokerId: Int,
 
   def shutdown() {
     info("shutting down")
-    for(handler <- runnables)
+    for (handler <- runnables)
       handler.shutdown
-    for(thread <- threads)
+    for (thread <- threads)
+    for (thread <- threads)
```

Comment:
Good catch, not sure what happened there. Thanks for the lightning quick review, the PR is still WIP, I should have mentioned it in the title.